### PR TITLE
Bump to the latest tc artifacts

### PR DESCRIPTION
**Pull request by chrisdennis**

### CODE REVIEW

File: clustered/server/entity/src/main/java/org/ehcache/clustered/server/store/ClusterTierPassiveEntity.java [hunk 1]
```diff
@@ -191,6 +191,7 @@ private EhcacheEntityResponse invokePassiveInternal(InvokeContext context, Ehcac
     return success();
   }
 
+  @SuppressWarnings("deprecation")
```
Comment (mathieucarbou) [inline clustered/server/entity/src/main/java/org/ehcache/clustered/server/store/ClusterTierPassiveEntity.java:194]: this is ok to leave all these deprecations ?

File: clustered/osgi-test/src/test/java/org/ehcache/osgi/OsgiTestUtils.java [hunk 2]
```diff
@@ -107,24 +107,21 @@ public static Cluster startServer(Path serverDirectory) throws IOException {
...
-      "--node-port=" + tsaPort,
-      "--node-group-port=" + tsaGroupPort,
-      "--node-log-dir=" + serverDirectory.resolve("logs"),
-      "--node-repository-dir=" + serverDirectory.resolve("repository"),
+      "--name=default-server",
+      "--hostname=localhost",
+      "--port=" + tsaPort,
+      "--group-port=" + tsaGroupPort,
+      "--log-dir=" + serverDirectory.resolve("logs"),
+      "--config-dir=" + serverDirectory.resolve("repository"),
```
Comment (mathieucarbou) [inline clustered/osgi-test/src/test/java/org/ehcache/osgi/OsgiTestUtils.java:124]: sorry I missed this part in the other PR. I hope you didn't spend a lot of time figuring these...

File: clustered/server/entity/src/main/java/org/ehcache/clustered/server/store/ClusterTierPassiveEntity.java (hunk 1 above)
Comment (chrisdennis) [reply clustered/server/entity/src/main/java/org/ehcache/clustered/server/store/ClusterTierPassiveEntity.java:194]: @myronkscott and I had a chat about them and he promises that he'll fix them in the next push.

**Comment by mathieucarbou** [comment]
Unable to merge, build status not reported (FYI @akomakom @chrisdennis you might want to force the merge).

image

**Pull request merged**